crates/holo_hash/Cargo.toml

```diff
@@ -16,10 +16,10 @@ normal = ["tracing"]
 [dependencies]
 serde = "1"
 serde_bytes = "0.11"
+blake2b_simd = {version = "0.5.10"}
```

Comment:
It's intentional that we don't want to include this hashing lib when including `holo_hash` in wasm code, because that bloats the wasm size and hashing is unnecessary in wasm. So I don't think this is an acceptable solution.